src/librbd/migration/QCOWFormat.cc

```diff
@@ -422,6 +431,7 @@ class QCOWFormat<I>::L2TableCache {
                    << "index=" << min_idx << " (loading)" << dendl;
     auto& l2_cache = l2_cache_entries[min_idx];
     l2_cache.l2_offset = l2_offset;
+    l2_cache.timestamp = ceph_clock_now();
```

Comment:
Just wondering, don't we want to update the timestamp when increasing the counter?

Reply:
:man_shrugging: if you look at QEMU's implementation, it's just a strict least-frequently-accessed cache with no decay. Adding the timestamp tie-breaker just here helps that one case of low count eviction but doesn't try to switch the cache to a LRU.

Reply:
ack